src/Microsoft.AspNetCore.SignalR.Core/HubConnectionContext.cs

```diff
@@ -560,6 +561,9 @@ private static class Log
             private static readonly Action<ILogger, Exception> _abortFailed =
                 LoggerMessage.Define(LogLevel.Trace, new EventId(8, "AbortFailed"), "Abort callback failed.");
 
+            private static readonly Action<ILogger, Exception> _clientTimeout =
+                LoggerMessage.Define(LogLevel.Debug, new EventId(9, "ClientTimeout"), "Client has not sent a message in too long. Closing connection.");
```

Comment:
Don't use "too long" ;)

Rephrase this (and if possible, put the timeout value in) as something like "Client timeout interval ({value}) elapsed without receiving a message. Closing connection."